sync/fetch_queue.go

```diff
@@ -48,8 +48,16 @@ func concatShortIds(items []types.Hash32) string {
 	return str
 }
 
+func (fq *fetchQueue) shutdownRecover() {
```

Comment:
the purpose of this hack is to hide the stack trace of the panic we currently might get due to flaws in the gracefull shutdown process.